ocs_ci/ocs/resources/pv.py

```diff
@@ -242,3 +242,25 @@ def get_node_pv_objs(sc_name, node_name):
         for pv_obj in pv_objs
         if pv_obj["metadata"]["labels"]["kubernetes.io/hostname"] == node_name
     ]
+
+
+def check_available_pvs(expected_available_pvs=1):
```

Comment:
We have verify_pvs_created in lso_helpers. Looks like you are trying to implement same. Please check and see if it can be reused,
https://github.com/red-hat-storage/ocs-ci/blob/83b8c9b71ceaf622ebd4a3ba012c9e6113e89029/ocs_ci/deployment/helpers/lso_helpers.py#L321